airbyte-integrations/connectors/source-snapchat-marketing/metadata.yaml

```diff
@@ -17,9 +17,9 @@ data:
   name: Snapchat Marketing
   remoteRegistries:
     pypi:
-      enabled: true
+      enabled: false
```

Comment:
why are we removing the connector from pypi? Is there another path to using it from pyairbyte?